src/BlazorUI/Bit.BlazorUI.Extras/Styles/Material/tokens.material.scss

```
@@ -8,22 +8,30 @@
 // differently is a global token the components already read (the pill button radius, the chip corner,
 // the switch geometry, the slider handle), so re-skinning the library stays a matter of retuning
 // tokens rather than of restyling components.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)